openlibrary/templates/account/create.html

```diff
@@ -63,7 +63,7 @@ <h1>$_("Sign Up")</h1>
                 </div>
             </div>
 
-        <small>$:_("By signing up, you agree to the Internet Archive's <a href='//archive.org/about/terms.php' target='_blank'>Terms of Service</a>.")</small>
+        <small>$:_("By signing up, you agree to the Internet Archive's <a href="//archive.org/about/terms.php" target="_blank">Terms of Service</a>.")</small>
```

Comment:
CAUTION!  The outer string is double-quoted and then inner strings are single-quoted.  Making all strings double-quoted is changing the meaning of the outer string.

Reply:
Fixed this is the latest PR